ingestion_tools/scripts/data_validation/fixtures/path.py

```diff
@@ -83,7 +83,13 @@ def frames_dir(run_dir: str, filesystem: FileSystemApi) -> str:
 def frames_files(frames_dir: str, filesystem: FileSystemApi) -> List[str]:
     """[Dataset]/[ExperimentRun]/Frames/[frame_name].mrc"""
     files = filesystem.glob(f"{frames_dir}/*")
-    return ["s3://" + file for file in files if "_gain" not in file]  # Exclude gain files
+    # Exclude gain files, add s3 prefix
+    refined_files = ["s3://" + file for file in files if "_gain" not in file]
+    # gain files are in the folder, but just no frames
+    if len(refined_files) == 0 and len(files) != 0:
+        pytest.skip(f"No frame files in frames directory: {frames_dir}")
+
+    return refined_files
```

Comment:
skip checks related to the frames files, if there's no frame files (and only gain files) in the Frames/ folder